api/utils/mwasHypothesisGenerator.mjs

```diff
@@ -0,0 +1,117 @@
+import { getMWASResults } from "./mwas.mjs"
+import { runCypherQuery } from '../clients/neo4j.mjs';
+import { runLLMCompletion } from '../clients/oai.mjs';
+
+export const getMwasHypothesis = async (bioprojects, targetVirusFamilies, pageStart, pageEnd, identifiers) => {
+
+    let model, role;
+    model = 'gpt4o';
+    if (model === 'gpt4o') {
+        role = 'system';
+    } else {
+        role = 'assistant'; // o1
+    }
+
+
+    const results = await getMWASResults(bioprojects, targetVirusFamilies, pageStart, pageEnd);
+    const TARGET_QUERY = "Eimeria"
```

Comment:
this shouldn't be hardcoded since it'll cause incorrect results, we can get query info by passing in `filters` which is done for other apis